var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -65,6 +65,8 @@ class Trilinos(CMakePackage):
     version('11.14.2', 'e7c3cdbbfe3279a8a68838b873ad6d51')
     version('11.14.1', 'b7760b142eef66c79ed13de7c9560f81')
 
+    variant('allpkgs', default=True,
```

Comment:
I wonder if a better idea would be to enable an `all_optional` variant which would correspond to `Trilinos_ENABLE_ALL_OPTIONAL_PACKAGES`. This way one would specify all top packages one needs, and all other may be dependent on.